src/hooks/airdrop/useAirdropTokensRefresh.ts

```diff
@@ -0,0 +1,42 @@
+import { useAirdropStore } from "@app/store/useAirdropStore";
+// import { useInterval } from "../useInterval";
+import { useCallback, useEffect } from "react";
+import { invoke } from '@tauri-apps/api/tauri';
+
+export function useAirdropTokensRefresh() {
+    const { airdropTokens, setAirdropTokens } = useAirdropStore();
+
+    // Handle refreshing the access token
+    const handleRefresh = useCallback(() => {
+        const expired = airdropTokens?.expiresAt && new Date(airdropTokens?.expiresAt * 1000) > new Date();
```

Comment:
Are you sure we need to multiply it by 1000?

Reply:
yes we need to multiply by 1000 so its in ms and not seconds

Reply:
okay